### DISCUSSION TITLE
loc idx out of upper bound err

### DISCUSSION THREAD

**Discussion by psn417** [deepmodeling/deepmd-kit#221] [Q&A] [answered]
Hello，
I meet this error, and then deepmd stop running
# warning: loc idx out of upper bound (ignored if warned for more than 10 times) 
# warning: loc idx out of upper bound (ignored if warned for more than 10 times) 
# warning: ghost idx out of upper bound (ignored if warned for more than 10 times) 
# warning: ghost idx out of upper bound (ignored if warned for more than 10 times) 
# warning: ghost idx out of upper bound (ignored if warned for more than 10 times) 
What's the meaning of this error?

**Comment by njzjz** [comment, marked as answer]
Some atoms are beyond the box. The warnings can be ignored, but if you want to remove warnings, try to wrap atoms into the box.
